hardhat.config.ts

```diff
@@ -61,11 +61,14 @@ function getRemoteNetworkConfig(
 
 function getHardhatConfig(): HardhatNetworkUserConfig {
   const networkConfig: HardhatNetworkUserConfig = {
-    hardfork: 'berlin',
-    blockGasLimit: 15000000,
+    //hardfork: 'berlin',
```

Comment:
Nit: if not using this can we delete it?

Reply:
sorry yeah I was playing with the config to see why my local test was failing